man/fi_info.1.md

```diff
@@ -92,28 +92,184 @@ the fi_info structure, see fi_getinfo(3).
 # USAGE EXAMPLES
 
 ```
-$ fi_info -n 30.0.11.1 -p usnic -t FI_EP_DGRAM
+$ fi_info -p verbs -t FI_EP_DGRAM
 ```
 
-This will respond with all fabric interfaces that can reach address 30.0.11.1
-using endpoint type FI_EP_DGRAM with the usNIC provider.
+This will respond with all fabric interfaces that use endpoint type
+FI_EP_DGRAM with the verbs provider.
+
+```
+fi_info \-c FI_MSG|FI_READ|FI_RMA
+```
+
+This will respond with all fabric interfaces that can support
+FI_MSG|FI_READ|FI_RMA capabilities.
+
+```
+fi_info \-m FI_CONTEXT|FI_LOCAL_MR
+```
+
+This will respond with all fabric interfaces that can support
+FI_CONTEXT|FI_LOCAL_MR modes.
```

Comment:
I would drop this example.  Mode bits are 'supported' but requirements of a provider.